#! /bin/sh /usr/share/dpatch/dpatch-run
## apache_log_path.dpatch by  <debian@onerussian.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixed apache path to the default in debian

@DPATCH@

diff -Naur -x '*~' fail2ban-0.7.1/config/filter.d/apache-auth.conf fail2ban-0.7.1.modified/config/filter.d/apache-auth.conf
--- fail2ban-0.7.1/config/filter.d/apache-auth.conf	2006-07-17 15:25:03.000000000 -0400
+++ fail2ban-0.7.1.modified/config/filter.d/apache-auth.conf	2006-09-05 10:59:07.000000000 -0400
@@ -17,7 +17,7 @@
 # Notes.:  logfile to monitor.
 # Values:  FILE  Default:  /var/log/httpd/access_log
 #
-logfile = /var/log/httpd/access_log
+logfile = /var/log/apache/error.log
 
 # Option:  timeregex
 # Notes.:  regex to match timestamp in Apache logfile. For TAI64N format,
